src/material/stepper/step-header.html

```diff
@@ -10,6 +10,10 @@
       [ngTemplateOutletContext]="_getIconContext()"></ng-container>
     <ng-container *ngSwitchDefault [ngSwitch]="state">
       <span *ngSwitchCase="'number'">{{_getDefaultTextForState(state)}}</span>
+      <span class="cdk-visually-hidden"
+            *ngIf="state == 'done' || state == 'edit'">
```

Comment:
Sorry for chim in, but wouldn't it be better to do it like this?

```suggestion
            *ngIf="state === 'done' || state === 'edit'">
```

Reply:
Shouldn't we be adding this text only when the state is `done`? `edit` means that the user is still editing it.

Reply:
Thanks @dev054!

@crisbeto @jelbourn I just updated this to have also separately have special text for "edit". I do mainly see the edit state so I figured it's worth having one too. Although your right its different than "done". Although I am not too sure if just saying editable makes sense. Thoughts?

Reply:
Maybe it should say something like "Current"? The only reason we have it as `edit` is because that's what the icon is called.

Reply:
We should already be capturing which step is selected/active by applying `aria-selected` to the `role="tab"` element, ni?

Reply:
@crisbeto @jelbourn Right now if a step is selected it will be announced because of the `aria-selected` attribute. Although this label is more of a way to help clarify that that tab has been started (and maybe even fully filled out) but you can come back and edit it